auth-api/src/auth_api/models/task.py

```diff
@@ -88,3 +88,10 @@ def find_by_task_for_account(cls, org_id, status):
         """Find a task instance that matches the provided id."""
         return db.session.query(Task).filter_by(relationship_id=org_id,
                                                 relationship_type=TaskRelationshipType.ORG.value, status=status).first()
+
+    @classmethod
+    def find_by_task_for_user(cls, org_id, status):
```

Comment:
name could be find_task_for_user or find_by_user_and_status ?

Reply:
I shall refactor as find_by_user_and_status